package/gluon-web-osm/Makefile

```
@@ -8,4 +8,14 @@ define Package/gluon-web-osm
   TITLE:=base src for OSM inlay
 endef
 
+define Package/gluon-config-mode-core/install
+       $(Gluon/Build/Install)
+
+ifdef CONFIG_GLUON_MINIFY
+       $(CP) ./javascript/gluon-web-osm.min.js $(1)/lib/gluon/web/www/static/
+else
+       $(CP) ./javascript/gluon-web-osm.js $(1)/lib/gluon/web/www/static/
```

We could go with something like this:
```suggestion
       $(CP) ./javascript/gluon-web-osm$(if $(CONFIG_GLUON_MINIFY),.min).js $(1)/lib/gluon/web/www/static/gluon-web-osm.js
```
Not entirely sure if I like that better or not though.

---

Thought about it, but didn't like it enough. If you want it I can change it.